Automattic/course
# Added a pattern for homepage

Implements https://github.com/Automattic/sensei/issues/6074

## Changes proposed

- Added a pattern with relevant font, template and styles for theme homepage.

## Testing instructions

- Create a new page
- In the editor, change the template of the page to **Page - No Title**
- Now clear if there's any block on the page and click on the inserter
- Click on patterns Tab
- Click on the dropdown for pattern categories and select "Sensei LMS" from there
- Make sure the **Landing Page** pattern is there, select it.
- Click on the pattern, save the page, and go to the frontend. Make sure the design matches

## Screenshots and Videos

**Desktop**

https://user-images.githubusercontent.com/6820724/196554308-2cae64a0-d273-43dc-8dcb-2c471386a8cc.mov

**Mobile**

https://user-images.githubusercontent.com/6820724/196554366-4cdb09f9-40d9-4260-91dd-3b971af23be0.mov
